cypress/e2e/smoke/teacher_student_sync_test.js

```diff
@@ -0,0 +1,79 @@
+import { visitQaSubtabsUnit } from '../../support/visit_params';
+import Canvas from '../../support/elements/common/Canvas';
+import ClueCanvas from '../../support/elements/common/cCanvas';
+import TextToolTile from '../../support/elements/tile/TextToolTile';
+import TeacherDashboard from '../../support/elements/common/TeacherDashboard';
+
+let canvas = new Canvas();
+let clueCanvas = new ClueCanvas();
+let textToolTile = new TextToolTile();
+let dashboard = new TeacherDashboard();
+
+context('Teacher/Student Sync Smoke Test', () => {
+  it('verifies teacher can see student work in readonly mode and changes sync', () => {
+    // Start as student 5
+    visitQaSubtabsUnit({ student: 5 });
+
+    // Add a text tile and enter some text
+    clueCanvas.addTile('text');
+    textToolTile.enterText('Student test message');
+    textToolTile.getTextEditor().should('contain', 'Student test message');
+
+    // Test sharing functionality
+    cy.log('verify share button');
+    clueCanvas.getShareButton().should('be.visible');
+    clueCanvas.getShareButton().should('have.class', 'private');
+    clueCanvas.shareCanvas();
+    clueCanvas.getShareButton().should('have.class', 'public');
+    clueCanvas.unshareCanvas();
+    clueCanvas.getShareButton().should('have.class', 'private');
+
+    // Publish the work
+    cy.log('verify publish button');
+    canvas.publishCanvas("investigation");
+    canvas.getPublishItem().should('exist');
+
+    // Now switch to teacher view
+    visitQaSubtabsUnit({ teacher: 1 });
+
+    // Test dashboard views
+    cy.log('verify dashboard views');
+    dashboard.switchView("Dashboard");
+    dashboard.switchWorkView('Published');
+    cy.wait(3000); // Wait for canvases to load
```

Comment:
Do you know if the wait is needed?
In most cases Cypress should automatically wait for an element to exist, and move on as soon as it does.